### PR TITLE
Audio analysis refactor

### PR DESCRIPTION
**Pull request by mattlianje**
Moved helper classes to lib, added location for audio files, added secrets.py, added our rds db url 

**TESTING INSTRUCTIONS:**

1. `$ cd audio-analysis`
2. Install a MySQL client (I am using DataGrips, but I guess MySQL workbench is fine)
3. Ask me for the `secrets.py` and copy it into the `audio-analysis` directory
4. Make sure `test_table` in your `climactic_test` DB has been deleted
5. `$ python main.py log`, then pick option `3`
6. Add the rds url to your MySQL client (like below) and you are good to query.
image

**TO DO:**
Set up read-only users for db.

### CODE REVIEW

**Review by tjlam — DISMISSED** [review]
LGTM, just add `__pycache__` to the gitignore

File: audio-analysis/.gitignore.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+__pycache__/
```
Comment (tjlam) [inline audio-analysis/.gitignore.txt]: I'm still seeing some pycache in the PR. You might have to do `**/__pycache__/` to cover everything

**Review by ewega — APPROVED** [review]
Looks good!!!!

File: audio-analysis/.gitignore.txt (hunk 1 above)
Comment (mattlianje) [reply audio-analysis/.gitignore.txt]: @tjlam oop mb mb) Removed manually and added to .gitignore. Should be good now.

**Pull request merged**